src/generation.rs

```diff
@@ -1028,6 +1038,9 @@ impl GenerationScope {
         export_raw_bytes_encoding_trait: bool,
         cli: &Cli,
     ) -> std::io::Result<()> {
+        // check it exists here to get clearer error message
```

Comment:
this is checked all the time instead of just when it's overridden as we still pull out the `Cargo.toml` files from here regardless.